### DISCUSSION TITLE
generated incorrect window post proof

### DISCUSSION THREAD

**Discussion by maweis1981** [filecoin-project/lotus#6328] [Storage Provider]
I have sealing sectors successful Proving yet, and onChain Now,
But when windowPoSt Failed, and all sectors meet Faulty.

version: lotus-miner version 1.8.0+mainnet+git.5c56017db.dirty

the log is here:

> 
> {"level":"info","ts":"2021-05-26T00:56:59.789+0800","logger":"storageminer","caller":"storage/wdpost_run.go:600","msg":"computing window post","batch":0,"elapsed":850.900546052}
> {"level":"error","ts":"2021-05-26T00:56:59.877+0800","logger":"storageminer","caller":"storage/wdpost_run.go:634","msg":"generated incorrect window post proof","post":[{"PoStProof":8,"ProofBytes":"jZkALoRMtXEabfEhVH4aPgoliEm1fDcT89p4WJsCO+R1mt2hgnlPgk2nEEuhuvIHgXGU3doKdnpZvKY95Es+6O7NewJ5j293kevKHpu8Yb1fsSWt3eCiHP5SiAb5AO2HCXZsklHI+zkptSPfWuYXm8m5DN+CqpEZ63hlmX1524LIBcONhc4hNi5mx3YICoWWkvOjyZ8T9Z64NLVwLEsrbSR1DRW3eq735UscpnNWze2FYicK/HdOHqfCfmHHGl1Q"}],"error":null}
> {"level":"warn","ts":"2021-05-26T00:56:59.882+0800","logger":"storageminer","caller":"storage/wdpost_run.go:226","msg":"Checked sectors","checked":1,"good":1}
> {"level":"info","ts":"2021-05-26T00:56:59.883+0800","logger":"storageminer","caller":"storage/wdpost_run.go:584","msg":"running window post","chain-random":"1RBwCWxrBBmR8ztcs3R5mhhWYV6fPvh6AAs/V0QYV+k=","deadline":{"CurrentEpoch":788379,"PeriodStart":788319,"Index":1,"Open":788379,"Close":788439,"Challenge":788359,"FaultCutoff":788309,"WPoStPeriodDeadlines":48,"WPoStProvingPeriod":2880,"WPoStChallengeWindow":60,"WPoStChallengeLookback":20,"FaultDeclarationCutoff":70},"height":"788379","skipped":0}
> 
> 

why "generated incorrect window post proof" ?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
what nvidia driver version are you using?
